Libraries/plist/Sources/UnixTime.cpp

```diff
@@ -16,8 +16,13 @@ using plist::UnixTime;
 void UnixTime::
 Decode(uint64_t in, struct tm &out)
 {
+#if _WIN32
+    time_t const t = in;
```

Comment:
(All of this code could likely use `std::chrono` instead of the C time APIs, which would avoid the issue as well as being much more type safe.)